spec/lib/save_applicant_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+require 'save_applicant'
+
+RSpec.describe SaveApplicant do
+  subject { described_class.new }
+
+  let(:valid_name) { "Rich" }
+  let(:valid_dob) { "1991-12-01" }
+
+  context "When given valid values" do
+    it "should save the model" do
+      applicant , success = subject.save_applicant(name: valid_name , date_of_birth: valid_dob)
+      expect(success).to eq true
+      expect(applicant.name).to eq valid_name
+
+    end
+
+  end
+
```

Comment:
There are some formatting issues across these changes. Rubocop is flagging them, and the CircleCI build is failing. You can run `bundle exec rubocop -a` to try and resolve them automatically (although you might have to fix some manually). 

It's generally a good idea to run `bundle exec rake` (or at least `bundle exec rubocop`) before committing, to avoid this.